src/components/organisms/events/events.scss

```
@@ -0,0 +1,69 @@
+@use "../../variables" as *;
+
+.events {
+  .events-border {
+    border: solid 1px $gray_gainsboro;
+    &:not(:last-of-type) {
+      border-bottom: none;
+    }
+    .events-href {
+      &:hover {
+        text-decoration: none;
+      }
+      .events-img {
+        display: none;
+      }
+      .events-theme-and-day-wrap {
+        padding: 10px 20px 16px;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        color: $black;
+        transition: 0.2s ease;
+        &:hover {
+          color: $orange_bistre;
+          cursor: pointer;
+        }
+        .events-theme {
+          margin-bottom: 8px;
+          font-size: $fontsize-m;
+          overflow: hidden;
+          white-space: nowrap;
+          text-overflow: ellipsis;
+          transition: 0.2s ease;
+        }
+        .events-day {
+          color: $gray_nobel;
+          font-size: $fontsize-xs;
+        }
+      }
+    }
+  }
+  .events-list-link-container {
+    text-align: right;
+    margin: 30px 0 35px;
+  }
+  @media (min-width: $breakpoint-lg) {
+    .events-border {
+      .events-href {
+        display: flex;
+      }
+      .events-img-hidden {
+        overflow: hidden;
+        width: 221px;
+        .events-img {
+          display: block;
+          height: 141px;
+          background-size: cover;
+          transition: all 0.5s ease;
+        }
+      }
+      .events-theme-and-day-wrap {
+        padding: 0 24px;
+      }
+    }
+  }
+  .events-border:hover .events-img {
+    transform: scale(1.2, 1.2);
```

拡大幅が同じであればこれでも問題ないと思います！
```suggestion
    transform: scale(1.2);
```